app/views/directives/osc-secrets.html

```diff
@@ -0,0 +1,62 @@
+<ng-form name="secretsForm" class="osc-secrets-form">
+  <div ng-repeat="pickedSecret in pickedSecrets">
+    <div class="form-group">
+      <div class="row picked-secret">
+        <div ng-class="type === 'Source Secret' ? 'col-sm-6' : 'col-sm-12'">
+          <label class="picker-label" ng-show="$index === 0">{{type | startCase}}</label>
+          <ui-select ng-model="pickedSecret.name">
+            <ui-select-match allow-clear="{{type === 'Source Secret' ? false : true}}" placeholder="Secret name">{{$select.selected}}</ui-select-match>
```

Comment:
Yeah well tat was my original idea but if we add the empty option it will be really hard to see, since its really small. Screen:
![rect4147](https://cloud.githubusercontent.com/assets/1668218/18547258/bf21eb2e-7b42-11e6-83ed-78db91beae6a.png)

But what we can do instead is to have a `X` button right from the picker, in the same way we have it for the "Source Secret" picker+input combination.